backend/src/apiserver/main.go

```diff
@@ -77,6 +77,17 @@ func main() {
 		}
 	}
 
+    logLevel := os.Getenv("LOG_LEVEL")
```

Comment:
I believe all the golang file are lint with tab not 4 white spaces. It still seems like your indent is a bit off.

Reply:
Done, rectified the indentation errors.